core/arch/arm/dts/hikey_sdp.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ *  Copyright (c) 2022, Linaro Limited
+ */
+
+/dts-v1/;
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		sdp@0x3e800000 {
+			compatible = "linaro,secure-heap";
```

Comment:
Where is this "compatible" documented?

Reply:
vendor prefix was taken from bindings documentation
https://www.kernel.org/doc/Documentation/devicetree/bindings/vendor-prefixes.yaml

secure-heap property here is added as a new feature.

Reply:
OK I understand that to mean that anything "linaro," can be documented elsewhere.
I guess it should still be documented somewhere. Since the kernel is going to use this it seems natural to document it in the kernel tree to. Or am I missing something?

Reply:
probably something like that in Documentation/devicetree/bindings/reserved-memory/linaro,secure-heap.yaml
# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
%YAML 1.2
---
$id: http://devicetree.org/schemas/reserved-memory/linaro,secure-heap.yaml#
$schema: http://devicetree.org/meta-schemas/core.yaml#

title: Linaro Secure DMABUF Heap

maintainers:
  - Olivier Masse <olivier.masse@nxp.com>

description: On Linaro OPTEE, the Secure Data Path feature
  need a reserved memory.

allOf:
  - $ref: "reserved-memory.yaml"

properties:
  compatible:
    const: linaro,secure-heap

  reg:
    description: region of memory reserved for OPTEE SDP feature

unevaluatedProperties: false

required:
  - compatible
  - reg

Reply:
done here
https://github.com/linaro-swg/linux/pull/101/commits/0c7ef9e676c79ca3f6b565944bb0d39f23750f6b

Reply:
Thanks